### DISCUSSION TITLE
Unraid Community App

### DISCUSSION THREAD

**Discussion by diegomontanaduarte** [emqx/emqx#9055] [Ideas]
I'm unsure if this right channel to request this, but it will be awesome to have EMQX MQTT broker Unraid App to easily deploy the docker container in our Unraid server, like we do with Mosquitto MQTT broker.

![image](https://user-images.githubusercontent.com/2682853/192317941-8bbab9fb-4cfc-4583-bf4e-ffb052cac326.png)

**Comment by markmghali** [comment]
Yes! this was be amazingly helpful!

**Comment by baldfox** [comment]
yeah definitely need this one. EMQX is awesome.
